server/@types/assessForEarlyReleaseApiClientTypes.ts

```diff
@@ -1,5 +1,14 @@
 import { components } from './assessForEarlyReleaseApiImport'
 
-export type OffenderSummary = components['schemas']['OffenderSummary']
-export type AssessmentSummary = components['schemas']['AssessmentSummary']
-export type InitialChecks = components['schemas']['InitialChecks']
+type ParsingDates<T, Fields> = {
+  [K in keyof T]: K extends Fields ? Date : T[K] extends object ? ParsingDates<T[K], Fields> : T[K]
+}
+
+export type _OffenderSummary = components['schemas']['OffenderSummary']
+export type OffenderSummary = ParsingDates<_OffenderSummary, 'hdced'>
```

Comment:
this basically rewrites the type to have certain fields as dates. 

It seems to work recursively to - so initial checks has `hdced` on a nested object and that also gets rewritten. 

Obviously the field have to be manually converted, see below..